doc/guides/portability/posix.rst

```diff
@@ -305,7 +305,7 @@ This is implemented as part of the minimal C library available in Zephyr.
     strtof(),
     strtoimax(),
     trtok(),
-    strtok_r(),
+    strtok_r(),+
```

Comment:
Please fix this misspelled `strtok` in the previous line.

Reply:
good catch :)

Reply:
actually please just delete it.

because minimal libc doesn't implement `strtok()`, just `strtok_r()`

Reply:
> actually please just delete it.

No, it should remain.  Only the functions with a + in the second column are actually supported.

Reply:
`trspn` is also wrong.  As is `trtoul`.  And several earlier ones.  And `canf`.  Seems to be limited to functions in the `s` section, though.

Reply:
> Please fix this misspelled strtok in the previous line.

Done.

Reply:
> > actually please just delete it.
> 
> No, it should remain. Only the functions with a + in the second column are actually supported.

Derp, missed that, thanks